M2/Macaulay2/packages/GradedLieAlgebras.m2

```diff
@@ -72,7 +72,6 @@ export {
     "mbRing",
     "minimalModel",
     "normalForm",
-    "sign",
```

Comment:
You removed it from here, but several places in the documentation link to `sign`, which now goes to an irrelevant node in Macaulay2Doc.